docs/hints.md

```diff
@@ -5,6 +5,94 @@ in go-ceph work together. This is not meant to cover every possible use
 case but are recorded here as a quick way to get familiar with these
 calls.
 
+
+## General
+
+### Finding an API
+
+The go-ceph project wraps existing APIs that are part of Ceph. There are two
+kinds of APIs that are wrapped. The first style of API is based on functions
+Ceph exports as client libraries in C. The three packages in go-ceph `cephfs`,
+`rados`, and `rbd` map to `libcephfs`, `librados`, and `librbd` respectively.
+
+The go-ceph packages that wrap Ceph C APIs follow a documentation convention
+that aims to make it easier to map between the APIs. In functions that are
+implemented using a certain C API function a line with the term `Implements:`
+will be followed by the C function's declaration - matching what can be found
+in the C library's header file. For example, if you knew you wanted to wrap the
+rbd function to get an image's metadata, `rbd_metadata_get`, you could search
+within the source code or https://pkg.go.dev/github.com/ceph/go-ceph/rbd for
+`rbd_metadata_get` which would lead you to the `GetMetadata` method of the
+`Image` type.
+
+The second style of API is based on functions implemented within Ceph services
+based on Ceph's "command" system. These functions are primarily accessed using
+the `ceph` command line command. Many of the functions within the ceph command
```

Comment:
```suggestion
the `ceph` command line. Many of the functions within the ceph command line 
```

Reply:
Same response as the other change to "command line command"

Reply:
and here:
```suggestion
the `ceph` command. Many of the functions within the ceph command
```

Reply:
Taken